Dockerfiles/gadget-default.Dockerfile

```diff
@@ -39,17 +42,36 @@ RUN cd /gadget/gadget-container && \
 	fi; \
 	make -j$(nproc) TARGET_ARCH=${TARGETARCH} gadget-container-deps
 
+# Execute BTFGen
+RUN set -ex; \
+	if [ "$ENABLE_BTFGEN" = true ]; then \
+		cd /gadget && \
+		make btfgen BPFTOOL=/tmp/btfhub/tools/bin/bpftool.x86_64 \
+			BTFHUB_ARCHIVE=/tmp/btfhub-archive/ OUTPUT=/tmp/btfs/ -j$(nproc); \
+	fi
+
```

Comment:
Why is this back? (It was moved to the host in https://github.com/inspektor-gadget/inspektor-gadget/pull/1228 to speed it up).